### DISCUSSION TITLE
Blocking set to only weekends, still works on weekdays.

### DISCUSSION THREAD

**Discussion by deloseka** [proginosko/LeechBlockNG#509] [Q&A] [answered]
This may be a relatively simple question, I'm new to the extension so I thought I'd ask. 

I'm setting it up so social media is completely blocked on weekends. Most social media will be blocked during the week, except for bsky.app and instagram.com for work, which will have a ten minute time limit. However, when I set up my blocks, despite having only one, with it set to only work on Saturday and Sunday, it still blocks my access during the week. 
I'm not exactly sure where I'm going wrong with setting this up, so a little guidance would be helpful! I've attached screenshots of what my first block looks like, I assume having the help to set up the first one will make it easier to set up the other two I'll need.

Thank you so much!
![1](https://github.com/user-attachments/assets/53a35053-71f8-4591-97ea-dde781b9083b)
![2](https://github.com/user-attachments/assets/79702c82-3ddf-4873-b748-0826b2e763dd)

**Comment by proginosko** [comment]
I don't see anything wrong with your settings on this page. Those sites should only be blocked on Sat/Sun. Can you show me the setting for the other two block sets? Alternatively, export your options to a text file (Options > General > Export Options) and email it to me at leechblockng@proginosko.com so I can take a closer look.

**Comment by proginosko** [comment]
Thanks for sending the file. Your settings look correct to me, so I'm perplexed as to why it would be blocking on other days than Sat/Sun.

Which time zone are you in? I wonder if your browser is reporting the wrong time to LeechBlock. Try this: Open a console (Ctrl+Shift+K) and enter `new Date()` followed by Enter. Does the console output the correct date, day, and time?

**Reply by deloseka** [reply]
My timezone is EST, looks like it's correct on FireFox too. Thank you again for helping me with this!

**Comment by proginosko** [comment, marked as answer]
So strange! You could try uninstalling and reinstalling LeechBlock, then importing your exported settings.

**Comment by deloseka** [comment]
Looks like uninstalling and reinstalling fixed it! Probably should've been the first thing I tried, haha! Thank you so much!
